components/script/dom/bindings/refcounted.rs

```diff
@@ -55,6 +55,7 @@ pub use self::dummy::LIVE_REFERENCES;
 /// A pointer to a Rust DOM object that needs to be destroyed.
 struct TrustedReference(*const libc::c_void);
 unsafe impl Send for TrustedReference {}
+unsafe impl Sync for TrustedReference {}
```

Comment:
I would revert this change. It's unclear to me that the value stored in `TrustedReference` is actually `Sync` which means it can be used safely from more than one thread at a time. By implement `unsafe impl Sync` here you are promising the compiler that is the case. As a reviewer I would need to look much more closely at this code to understand if that's the case. Since it is unrelated to the other change which I do understand, please just revert it for now.

Reply:
I delete this change, and push it to my branch. I am new to Contribute, sorry if I made some mistakes.